src/Integration.Vsix.UnitTests/ErrorList/SonarErrorListDataSource_IssueLocationStoreTests.cs

```diff
@@ -264,6 +250,38 @@ public void RefreshOnBufferChanged_FactoriesWithMatchingFilesUpdatedAndErrorList
             CheckSinkNotifiedOfChangeToFactories(sinkMock2, factoryWithMatch1, factoryWithMatch2);
         }
 
+        [TestMethod]
+        public void RefreshImpl_SomeFactoriesWithMatchingFiles_RaisesSingleIssueChangeEventContainingExpectedFiles()
```

Comment:
I see this is only for Refresh, not RefreshOnBufferChanged. Why not use your previous way of testing both with the callback? It should just "work" since there should be only one factory, but it's worth asserting.

Reply:
The existing tests check that the event is raised/not raised in various circumstances (and `RefreshOnBufferChanged` should never raise the event).

This test is to specifically check the contents of the event args. Since `RefreshOnBufferChanged` doesn't raise the event, I thought it would be confusing to include it.